index.bs

```diff
@@ -1610,7 +1876,255 @@ level APIs. The list above includes interface classes for which such high level
 APIs exist and provide greater protection for user privacy and security than
 low level access through this API would.
 
-## Endpoints ## {#endpoints}
+<div algorithm="find if the interface is claimed">
+    To <dfn>find if the interface is claimed</dfn> with the given |interface|,
+    perform the following steps:
+    1. Let |configuration| be |interface|.{{USBInterface/[[configuration]]}}.
+    1. Let |device| be |configuration|.{{USBConfiguration/[[device]]}}.
+    1. If |configuration| is not the same as the result of <a>finding the current configuration</a>
+        with |device|, return <code>false</code>.
+    1. Let |interfaceIndex| be the result of <a>finding the interface index</a>
+        with |interface|.{{USBInterface/[[interfaceNumber]]}} and |configuration|.
+    1. <a>Assert</a>: |interfaceIndex| is not equal to <code>-1</code>.
+    1. Return |device|.{{USBDevice/[[claimedInterface]]}}[|interfaceIndex|].
+</div>
+
+<div algorithm="find the alternateInterface for the current alternate setting">
+    To <dfn>find the alternateInterface for the current alternate setting</dfn> with the given |interface|,
```

Comment:
```suggestion
<div algorithm="find the alternate interface for the current alternate setting">
    To <dfn>find the alternate interface for the current alternate setting</dfn> with the given |interface|,
```

Reply:
Updated in https://github.com/WICG/webusb/pull/212/commits/5aaf696938ed1dc073f8cc8da8c809e9e5019f41.